src/AddItemPage/FileInput.js

```diff
@@ -5,9 +5,16 @@ import "./FileInput.css";
 
 function FileInput({ title }) {
   const [imagePreviewUrl, setImagePreviewUrl] = useState("");
+  const [error, setError] = useState("");
 
   const handleImageChange = (event) => {
     const file = event.target.files[0];
+    if (imagePreviewUrl) {
+      setError("*이미지 등록은 최대 1개까지 가능합니다.");
+      setTimeout(() => setError(""), 3000); // 3초 후 경고 메시지 해제
+      return;
+    }
+
```

Comment:
빨리 return 할 수 있는 부분은 최대한 빨리 리턴하시는 걸 추천드려요!

```suggestion
    if (imagePreviewUrl) {
      setError("*이미지 등록은 최대 1개까지 가능합니다.");
      setTimeout(() => setError(""), 3000); // 3초 후 경고 메시지 해제
      return;
    }
    
      const file = event.target.files[0];

```